code/M3D/m3d.h

```diff
@@ -88,7 +91,7 @@ typedef uint16_t M3D_INDEX;
 #else
 #define _inline
 #define _pack
-#define _unused
+#define _unused __pragma(warning(suppress:4100))
```

Comment:
One hint: you can also get aware of this warning by this:
`void foo(int i) {}`

to

`void foo(int ) {}`

Reply:
This is only valid in C++ but not in C.

Reply:
turol is right, that wouldn't compile under C.

Reply:
We can add just a cpp preprocessor step to make sure that it works only when we are compiling this as c++. And then this issue is resolved from my point of view.

Reply:
no need for the ifdef, works in ANSI C too.
https://docs.microsoft.com/en-us/cpp/preprocessor/pragma-directives-and-the-pragma-keyword?view=vs-2019&viewFallbackFrom=vs-2013